yt/utilities/tests/test_interpolators.py

```diff
@@ -152,3 +153,63 @@ def test_get_vertex_centered_data():
     ds = fake_random_ds(16)
     g = ds.index.grids[0]
     g.get_vertex_centered_data([("gas", "density")], no_ghost=True)
+
+
+_lin_interpolators_by_dim = {
+    1: lin.UnilinearFieldInterpolator,
+    2: lin.BilinearFieldInterpolator,
+    3: lin.TrilinearFieldInterpolator,
+    4: lin.QuadrilinearFieldInterpolator,
+}
+
+
+@pytest.mark.parametrize("ndim", list(range(1, 5)))
+def test_table_override(ndim):
+    sz = 8
+
+    random_data = np.random.random((sz,) * ndim)
+
+    field_names = "xyzw"[:ndim]
+    slc = slice(0.0, 1.0, complex(0, sz))
+    fv = dict(zip(field_names, np.mgrid[(slc,) * ndim]))
+    boundaries = (0.0, 1.0, 0.0, 1.0, 0.0, 1.0, 0.0, 1.0)[: ndim * 2]
+
+    interp_class = _lin_interpolators_by_dim[ndim]
+    interpolator = interp_class(random_data, boundaries, field_names, True)
+    assert_array_almost_equal(interpolator(fv), random_data)
+    table_2 = random_data * 2
+    assert_array_almost_equal(interpolator(fv, table=table_2), table_2)
+
+    # check that we can do it without storing the initial table
```

Comment:
This comment would be even more helpful if it also mentioned that `store_table=True` is implicit in the previous example as it's the default value.

Reply:
added more comments above